res/lang/programme/programme_en.txt

```diff
@@ -76,28 +76,28 @@ PROGRAMME_GENRE_EVENT_SPORT                = Sport
 PROGRAMME_GENRE_EVENT_SHOWBIZ              = Showbiz
 PROGRAMME_GENRE_EVENT_MUSIC                = Music and Singing
 PROGRAMME_GENRE_FEATURE                    = Feature
-PROGRAMME_GENRE_FEATURE_YELLOWPRESS        = Yellow Press
+PROGRAMME_GENRE_FEATURE_YELLOWPRESS        = Tabloid journalism
```

Comment:
Not sure about this. "Yellow Press" is more "Klatsch" (celebrities etc) than "we have the sensations to tell!" things (bild-style).

Reply:
Would "Gossip" be better?

Reply:
@scr0llbaer9000 any meaning?
Gossip is for me the term used when doing "small talk" about your neighbours.

What I am talking about is more the "Explosiv", "Brisant", "Taff"-style of magazines/shows. So they kinda focus on scandals of "celebrities" and less on "Politicians having sold their soul".

Reply:
Yeah, but we use "gossip" for that too, like "Gossip magazines".

Reply:
btw ... while overhauling some achievements (for my stationmap overhaul PR).

```XML
		<achievement id="tvt-gameachievement-broadcastNewsShow-showbiz1" creator="5578" created_by="Ronny">
			<title>
				<de>Klatschreporter</de>
				<en>Gossip reporter</en>
			</title>
			<!-- kein Text = automatisches Generieren aus Aufgabentext + Belohnung -->
			<text />
			<tasks>
				<task id="tvt-gameachievement-task-broadcastNewsShow-showbiz1" creator="5578" created_by="Ronny">
					<title>
						<de>Sende 3 Showbiz-Nachrichten</de>
						<en>Broadcast 3 showbiz news</en>
					</title>
					<text>
						<de>Sende 3 Showbiz-Nachrichten in einer Nachrichtensendung.</de>
						<en>Broadcast 3 showbiz news in one single news broadcast.</en>
					</text>
					<data type="BroadcastNewsShow" genre1="1" genre2="1" genre3="1" />
				</task>
			</tasks>
```
There I used "gossip" in connection to "showbiz" news

Reply:
I've changed it to Gossip then in the latest commit.